api/tests/routes/backoffice_v3/offerers_test.py

```diff
@@ -1861,3 +1861,132 @@ def send_request(authenticated_client, offerer_id, url, form_data=None):
     form = {"csrf_token": g.get("csrf_token", ""), **form_data}
 
     return authenticated_client.post(url, form=form)
+
+
+class ListOffererTagsTest:
+    class UnauthorizedTest(unauthorized_helpers.UnauthorizedHelper):
+        endpoint = "backoffice_v3_web.offerer_tag.list_offerer_tags"
+        endpoint_kwargs = {"offerer_id": 1}
+        needed_permission = perm_models.Permissions.MANAGE_PRO_ENTITY
+
+    def test_list_offerer_tags(self, authenticated_client):
+        # given
+        category = offerers_factories.OffererTagCategoryFactory(label="indépendant")
+        offerer_tag = offerers_factories.OffererTagFactory(
+            name="scottish-tag",
+            label="Taggy McTagface",
+            description="FREEDOOOOOOOOOM",
+            categories=[category],
+        )
+
+        # when
+        with assert_no_duplicated_queries():
+            response = authenticated_client.get(url_for("backoffice_v3_web.offerer_tag.list_offerer_tags"))
+
+        # then
+        assert response.status_code == 200
+        rows = html_parser.extract_table_rows(response.data)
+        assert len(rows) == 1
+        assert rows[0]["Nom"] == offerer_tag.name
+        assert rows[0]["Libellé"] == offerer_tag.label
+        assert rows[0]["Description"] == offerer_tag.description
+        assert rows[0]["Catégories"] == category.label
+
+
+class UpdateOffererTagTest:
+    class UnauthorizedTest(unauthorized_helpers.UnauthorizedHelperWithCsrf):
+        method = "post"
+        endpoint = "backoffice_v3_web.offerer_tag.update_offerer_tag"
+        endpoint_kwargs = {"offerer_tag_id": 1}
+        needed_permission = perm_models.Permissions.MANAGE_PRO_ENTITY
+
+    def test_update_offerer(self, legit_user, authenticated_client):
+        offerer_tag_not_to_edit = offerers_factories.OffererTagFactory()
+        category = offerers_factories.OffererTagCategoryFactory(label="Le travail c'est la santé")
+        offerer_tag_to_edit = offerers_factories.OffererTagFactory(
+            name="tag-you-are-it",
+            label="C'est toi le loup",
+            description="Le jeu du loup c'est le 'tag' en anglais hihi",
+        )
+
+        new_name = "very-serious-tag"
+        new_label = "Tag très sérieux"
+        new_description = "Pas le temps de jouer"
+        new_categories = [category.id]
+
+        base_form = {
+            "name": new_name,
+            "label": new_label,
+            "description": new_description,
+            "categories": new_categories,
+        }
+        response = self.update_offerer_tag(authenticated_client, offerer_tag_to_edit, base_form)
+        assert response.status_code == 303
+
+        # Test redirection
+        expected_url = url_for("backoffice_v3_web.offerer_tag.list_offerer_tags", _external=True)
+        assert response.location == expected_url
+
+        response = authenticated_client.get(expected_url)
+
+        assert f"{new_name}" in html_parser.content_as_text(response.data)
```

Comment:
Détail : Je pense que tu peux utiliser `html_parser.extract_table_rows(response.data)` comme au-dessus, ce sera mieux typé, comme de toute façon tu n'avais aucun tag il n'y en a a priori qu'un seul dans la table.
Ou sinon `text = html_parser.content_as_text(response.data)` puis `... in text` pour ne faire qu'une fois le _parsing_ du html.